### ISSUE TITLE
[Feature]: When no bingo is backed up, the result of the backup operation should be set to fail

### ISSUE THREAD

**Issue by zhuwenxing** [zilliztech/milvus-backup#47]
### Is your feature request related to a problem? Please describe.

see https://github.com/zilliztech/milvus-backup/issues/45

The current backup operation does not have a binlog but will show a warning log, but the result will still be success, this backup is meaningless because the restored collection will be empty. This success result will be misleading to the user.

**Comment by minminggu** [comment]
Yes , I use milvus-backup(version 0.3) to backup milvus 2.2.10, the result is success but no binlog, only have four jason files in meta.
